ibc-apps/ics721-nft-transfer/types/src/events.rs

```diff
@@ -1 +1,209 @@
 //! Defines Non-Fungible Token Transfer (ICS-721) event types.
+use ibc_core::channel::types::acknowledgement::AcknowledgementStatus;
+use ibc_core::primitives::prelude::*;
+use ibc_core::primitives::Signer;
+use ibc_core::router::types::event::ModuleEvent;
+
+use super::Memo;
+use crate::{PrefixedClassId, TokenIds, MODULE_ID_STR};
+
+const EVENT_TYPE_PACKET: &str = "nft_packet";
+const EVENT_TYPE_TIMEOUT: &str = "timeout";
+const EVENT_TYPE_CLASS_TRACE: &str = "nft_class_trace";
```

Comment:
```suggestion
const EVENT_TYPE_CLASS_TRACE: &str = "class_trace";
```

If wanna keep it the same (Particulary from the relayer's point of view) as the ref implementation: https://github.com/bianjieai/nft-transfer/blob/b57ce734069ee36e8024fa480111e115e959138e/types/events.go#L9